### PR TITLE
Add a few iCalendar properties

### PR DESCRIPTION
**Pull request by tcitworld**
* [Status](https://tools.ietf.org/html/rfc5545#section-3.8.1.11), which matches to string
* [Categories](https://tools.ietf.org/html/rfc5545#section-3.8.1.2), which matches to a list of strings
* [Class](https://tools.ietf.org/html/rfc5545#section-3.8.1.3), which matches to string
* [Comment](https://tools.ietf.org/html/rfc5545#section-3.8.1.4), which is a simple text field
* [Geo](https://tools.ietf.org/html/rfc5545#section-3.8.1.6), which maps to a tuple with latitude and longitude

### CODE REVIEW

File: lib/icalendar/util/deserialize.ex [hunk 1]
```diff
@@ -132,6 +162,15 @@ defmodule ICalendar.Util.Deserialize do
     to_date(date_string, %{"TZID" => "Etc/UTC"})
   end
 
+  defp to_geo(geo) do
+    geo
+    |> desanitized()
+    |> String.split(";")
+    |> Enum.map(fn x -> Float.parse(x) end)
+    |> Enum.map(fn {x, _} -> x end)
```
Comment (tcitworld) [inline lib/icalendar/util/deserialize.ex:170]: I guess this may be improved

File: lib/icalendar/util/deserialize.ex [hunk 2]
```diff
@@ -85,6 +85,40 @@ defmodule ICalendar.Util.Deserialize do
...
+    acc
+  ) do
+    %{acc | comment: desanitized(comment)}
+  end
+  def parse_attr(
+    %Property{key: "STATUS", value: status},
+    acc
+  ) do
+    %{acc |
+      status: status |> desanitized() |> String.downcase() |> String.to_atom()
```
Comment (lpil) [inline lib/icalendar/util/deserialize.ex]: Can we remove the `String.to_atom/1` please? Turning random input to atoms is an easy way to leak atoms which can crash the VM.

File: lib/icalendar/util/deserialize.ex (hunk 1 above)
Comment (lpil) [reply lib/icalendar/util/deserialize.ex:170]: Fine by me :)

**Review by lpil — CHANGES_REQUESTED** [review]

**Comment by lpil** [comment]
Thanks @tcitworld, this is great. Left some comments :)

**Comment by tcitworld** [comment]
Fixed :)

File: test/icalendar/event_test.exs [hunk 3]
```diff
@@ -155,7 +155,7 @@ defmodule ICalendar.EventTest do
 
   test "ICalendar.to_ics/1 with class" do
     ics = %Event{
-      class: :private
+      class: "private"
```
Comment (lpil) [inline test/icalendar/event_test.exs]: Why have these been changed?

Comment (tcitworld) [reply test/icalendar/event_test.exs]: My bad

**Review by lpil — APPROVED** [review]
Thanks!

**Comment by lpil** [comment]
Published a new version, thanks!

**Comment by tcitworld** [comment]
thank you

**Pull request merged**